modules/serverless-rn-1-33-0.adoc

```diff
@@ -16,15 +16,21 @@
 * {ServerlessProductName} now uses Kourier 1.12.
 * {ServerlessProductName} now uses Knative (`kn`) CLI 1.12.
 * {ServerlessProductName} now uses Knative for Apache Kafka 1.12.
-* The `kn func` CLI plugin now uses `func` 1.13.
+* The `kn func` CLI plugin now uses `func` 1.14.
 
 * OpenShift Serverless on ARM64 is now available as Technology Preview.
 * The `NamespacedKafka` annotation is now deprecated. Use the standard Kafka broker without data plane isolation instead.
 
 * When enabling the automatic `EventType` auto-creation, you can now easily discover events available within the cluster. This functionality is available as a Developer Preview.
 
+* {FunctionsProductName} now supports Go function container and is now available as a Technology Preview feature.
```

Comment:
This is only supported using `pack` community builder. So I do think it shouldn't be listed as TP. The only addition we have for Go functions are the `--container=false` experimental option (scaffolding) by now. 
/cc @nainaz